src/NUnitConsole/nunit3-console/Program.cs

```diff
@@ -64,6 +65,20 @@ public static int Main(string[] args)
                 return ConsoleRunner.INVALID_ARG;
             }
 
+            if (!string.IsNullOrEmpty(Options.ConsoleEncoding))
+            {
+                try
+                {
+                    Console.OutputEncoding = Encoding.GetEncoding(Options.ConsoleEncoding);
```

Comment:
Is the root of the problem that the console.encoding is UTF-8, but you need an ASCII encoding (iso-885901) for your scenario?